torchaudio/prototype/rnnt_pipeline.py

```diff
@@ -0,0 +1,149 @@
+from typing import Callable, List, Tuple
+from dataclasses import dataclass
+import json
+import math
+import os
+import sys
+import torch
+
+import sentencepiece as spm
+import torchaudio
+from torchaudio._internal import download_url_to_file, load_state_dict_from_url
+from torchaudio.prototype import RNNT, RNNTBeamSearch, emformer_rnnt_base
+
+
+__all__ = []
+
+
+_BASE_MODELS_URL = "https://download.pytorch.org/torchaudio/models"
+_BASE_PIPELINES_URL = "https://download.pytorch.org/torchaudio/pipeline-assets"
+
+
+def _download_asset(asset_path: str):
+    dst_path = f"{os.getcwd()}/_assets/{asset_path}"
```

Comment:
Can you use [`torch.hub.get_dir()`](https://github.com/pytorch/pytorch/blob/12afe2bb84f56780c299a46fbde34639720ba99c/torch/hub.py#L245-L261) as the base directory path, so that it will be configurable via environment variable. 

See also

https://github.com/pytorch/audio/blob/1b17b0116548f60ccb55ef97d09afa02bca7aa26/torchaudio/pipelines/_tts/utils.py#L174-L180